### PR TITLE
Initialize values

### PR DESCRIPTION
**Pull request by m-kuhn**
Fixes a crash with the fill ring tool

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-a2a33ffea61f318d92840b8084cd32b447d66c8a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-a2a33ffea61f318d92840b8084cd32b447d66c8a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-a2a33ffea61f318d92840b8084cd32b447d66c8a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-a2a33ffea61f318d92840b8084cd32b447d66c8a-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-cb4d0d3064cd0916ccaf5b8535f867ba2c95e435-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-cb4d0d3064cd0916ccaf5b8535f867ba2c95e435-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-cb4d0d3064cd0916ccaf5b8535f867ba2c95e435-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-cb4d0d3064cd0916ccaf5b8535f867ba2c95e435-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1e3cbe41ec401a124b4f251e5dfe8d2c04b6c27a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1e3cbe41ec401a124b4f251e5dfe8d2c04b6c27a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1e3cbe41ec401a124b4f251e5dfe8d2c04b6c27a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1e3cbe41ec401a124b4f251e5dfe8d2c04b6c27a-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1db4ad2da3f81efe91d996ef721cb3562bbebf9d-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1db4ad2da3f81efe91d996ef721cb3562bbebf9d-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1db4ad2da3f81efe91d996ef721cb3562bbebf9d-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-918-1db4ad2da3f81efe91d996ef721cb3562bbebf9d-x86_64.apk)

File: src/core/attributeformmodelbase.h [hunk 1]
```diff
@@ -96,9 +96,9 @@ class AttributeFormModelBase : public QStandardItemModel
      */
     QgsEditorWidgetSetup findBest( int fieldIndex );
 
-    FeatureModel *mFeatureModel;
-    QgsVectorLayer *mLayer;
-    QgsAttributeEditorContainer *mTemporaryContainer;
+    FeatureModel *mFeatureModel = nullptr;
```
Comment (signedav) [inline src/core/attributeformmodelbase.h:99]: This makes perfectly sense :slightly_smiling_face:

**Pull request merged**